images/server/install-packages.sh

```diff
@@ -10,20 +10,28 @@ need_curl() {
     dnf install --setopt=install_weak_deps=False -y /usr/bin/curl
 }
 
-get_custom_repo() {
-    url="$1"
-    fname="$(basename "$url")"
-    dest="/etc/yum.repos.d/${fname}"
-    while [ -e "$dest" ]; do
-        u="$(echo "${url}" | sha256sum | head -c12)"
-        if [ -z "$u" ]; then
-            echo "failed to uniquify repo file name"
-            exit 2
+get_custom_repos() {
+    if [[ -n $1 ]]; then
+        urls=$1
+        need_curl
+        if [[ $(wc -w <<< "$1") -eq 1 || $ceph_from_custom -ne 1 ]]; then
```

Comment:
Why we just install the first repo if `ceph_from_custom` is not specified ? if this option is not specified, only one additional repo can be installed ?

Also, if `ceph_from_custom` is specified and we have a single custom repo, why we install the standard ceph repo ? shouldn't it be installed from one of the custom repos ?